pixeltable/utils/av.py

```diff
@@ -5,6 +5,14 @@
 
 from pixeltable.env import Env
 
+# format -> (codec, extension)
+audio_format_defaults: dict[str, tuple[str, str]] = {
```

Comment:
Good idea to refactor this. As long as we're moving it here, maybe rename it `AUDIO_FORMATS`?